account_invoice_pricelist/models/account_invoice.py

```diff
@@ -25,6 +25,7 @@ def _onchange_partner_id_account_invoice_pricelist(self):
     def button_update_prices_from_pricelist(self):
         for inv in self.filtered(lambda r: r.state == 'draft'):
             inv.invoice_line_ids.filtered('product_id').update_from_pricelist()
+        self.filtered(lambda r: r.state == 'draft').compute_taxes()
```

Comment:
Without this line taxes are wrong.
In production since one week
ping @legalsylvain @rvalyi